docs/api/actions.md

```diff
@@ -226,20 +226,29 @@ await element(by.id('pickerView')).setColumnToValue(2, "Hello World");
 
 > **Note:** When working with date pickers, you should always set an explicit locale when launching your app in order to prevent flakiness from different date and time styles. See [here](device.md#9-languageandlocalelaunch-with-a-specific-language-andor-local-ios-only) for more information.
 
-### `setDatePickerDate(dateString, dateFormat)`  iOS only
+### `setDatePickerDate(dateString, dateFormat)`
 
 Sets the element’s date to the specified date string, parsed using the specified date format.
 
-The specified date string is converted by the system to an [`NSDate`](https://developer.apple.com/documentation/foundation/nsdate) object, using [`NSDateFormatter`](https://developer.apple.com/documentation/foundation/dateformatter) with the specified date format, or [`NSISO8601DateFormatter`](https://developer.apple.com/documentation/foundation/iso8601dateformatter) in case of ISO 8601 date strings. If you use JavaScript’s [Date.toISOString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) or otherwise provide a valid ISO 8601 date string, set the date format to `"ISO8601"`, which is supported as a special case.
+In case of dateFormat set to `"ISO8601"`, the specified date string is converted using JavaScript’s `new Date(dateString)` on Android and [`NSISO8601DateFormatter`](https://developer.apple.com/documentation/foundation/iso8601dateformatter) on iOS. You can use [Date.toISOString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) to format a JavaScript date accordingly.
```

Comment:
Thanks, I'll re-read your docs addendum tomorrow. Sorry, had no time today.

Reply:
No hurry. Feel free to completely rewrite this if neccessary. I am not that good in writing docs.

Reply:
Rewrote some parts.